app/assets/stylesheets/organisms/_form-cards.scss

```diff
@@ -20,6 +20,7 @@
 .form-card--long {
   .form-question {
     font-size: 1.9rem;
+    line-height: 2.5rem;
```

Comment:
I did check through other forms with from-card--long class and they all have shortly named labels with is probably why the line-height adjustment has not been needed until now.